core/src/main/scala/kafka/coordinator/group/CoordinatorLoaderImpl.scala

```diff
@@ -187,13 +188,16 @@ class CoordinatorLoaderImpl[T](
           }
 
           val endTimeMs = time.milliseconds()
+          info(s"Finished loading offsets and group metadata from $tp "
+            + s"in ${endTimeMs - startTimeMs} milliseconds, of which " +
+            s"$schedulerTimeMs milliseconds was spent in the scheduler.")
```

Comment:
We already have a similar log in CoordinatorRuntime. I wonder if we should remove this one and extend the existing one. What do you think?